### ISSUE TITLE
Data Preparation

### ISSUE THREAD

**Issue by yitianlian** [embodied-generalist/embodied-generalist#10]
About the Cap3D_pcs_pt dataset, I find that the dataset is about 200GB. Should I need to download them all?

**Comment by huangjy-pku** [comment]
Yes if you want to reproduce the alignment stage training. The 200G dataset includes all point clouds for the object captions of 660k instances.
